### DISCUSSION TITLE
Ask the community for help

### DISCUSSION THREAD

**Discussion by izzycsy** [DeFiCh/app#678] [Q&A]
#### Is your question related to a problem? Please describe.
_A clear and concise description of what the question is. Eg. I'm curious to know [...]_

#### Additional context
_Add any other context or screenshots about the question posted here._

**Comment by RIPAC** [comment]
I noticed slippage I believe, allot. I went from 7K USDT and equal DFI to 6,500 USDT, how can this be so far off? I hadn't removed any coins. Is this slippage exactly? Will it ever go up in my USDT holdings? I also lost some of my BTC but not that much. Any insight is helpful because that's 500 gone and it doesn't feel good. I thought it would be safe since it's a DEX. Shouldn't it record exactly how much is held?

**Reply by uzyn** [reply]
Please read:

- https://defichain.ghost.io/the-7-most-frequently-asked-questions-about-liquidity-mining/
- https://defichain.ghost.io/defichain-impermanent-loss-risk-explained/

**Comment by mirbach** [comment]
I'm curious to know what is the timeframe for adding hardware wallet support like Trezor and Ledger?

**Comment by m1at1** [comment]
Hi: I've been using the beta BTC wallet in the linux app (on Manjaro Linux). 

Following a crash of my computer, i re-indexed the wallet, and the beta BTC wallet went to zero. However, no additional transactions have occurred. 

How can I report this to whoever is developing this part of the app? Is there a way to fix it? If not, is there a way to verify (perhaps in the console) that the BTC is still "there"?

**Reply by izzycsy** [reply]
Kindly explain your issue here: https://github.com/DeFiCh/app/issues/1043

**Comment by Oldman12345** [comment]
I don't understand the permanent change in "interest" coming from block rewards. The total of block rewards is fixed due to a DFIP - ok. The crypto pools have a fixed allocation while the token pools were reallocated after each new emission. In my understanding the (for example) btc pool has a fixed DFI reward allocation which is allocated to all liquidity providers. Okay, in my understanding that leads to decreasing rewards per unit when the total of LM unit grows. That should be easy math, but lets say today at 14.43 UTC the volume in the btc pool was 292793705 with a APR of 49,81 (resulting of reward without commissions) but than at 19.10 UTC the volume was 297215926 which means an increase of 1.51 per cent. In my understanding the rewards should decrease in the same relation, which means they should be at 49,06 per cent - but they were at 49,73 % (lower but not in the same relation). Why is that? I have no explaination

**Comment by pcolamar** [comment]
Hi,
  I am using the DEFI desktop linux app.
  How is it possible to download (csv) all the transactions included the rewards ?
If I use the standard interface, no matter how I toggle the include rewards"switch", they don't get in the csv.
They are just visible on the screen.
Any hint?
Thanks
Palmer
